cli/cmd/sync2.go

```diff
@@ -15,14 +15,14 @@ import (
 
 func syncConnectionV2(ctx context.Context, cqDir string, sourceClient *clients.SourceClient, sourceSpec specs.Source, destinationsSpecs []specs.Destination, uid string, noMigrate bool) error {
 	var err error
-	destinationNames := make([]string, len(destinationsSpecs))
+	destinationStrings := make([]string, len(destinationsSpecs))
```

Comment:
zerolog supports `Stringers` and `Stringer`, however `Stringers` is not very useful as we need to create that array manually (we can't cast to an array directly)